images/win/scripts/Installers/Validate-Go.ps1

```diff
@@ -54,10 +54,13 @@ _Environment:_
 $SoftwareName = "Go (x64)"
 $Description = New-Object System.Text.StringBuilder
 $goVersionsToInstall = $env:GO_VERSIONS.split(",")
+$refsJson = Invoke-RestMethod "https://api.github.com/repos/golang/go/git/refs/tags"
```

Comment:
We need the same changes for `Validate-Go.ps1`. Just improve `Get-GoVersion` to get the part of the path.
For example, we search go 1.13, just find the folder that can be `C:\Go1.13.2` for example.